examples/python/pincell_depletion/restart_depletion.py

```diff
@@ -0,0 +1,85 @@
+import openmc
+import openmc.deplete
+import numpy as np
+import matplotlib.pyplot as plt
+
+###############################################################################
+#                      Simulation Input File Parameters
+###############################################################################
+
+# OpenMC simulation parameters
+batches = 100
+inactive = 10
+particles = 1000
+
+# Depletion simulation parameters
+time_step = 1*24*60*60 # s
+final_time = 5*24*60*60 # s
+time_steps = np.full(final_time // time_step, time_step)
+
+chain_file = './chain_simple.xml'
+power = 174 # W/cm, for 2D simulations only (use W for 3D)
+
+###############################################################################
+#                      Load previous simulation results
+###############################################################################
+
+# Load geometry from statepoint
+statepoint = 'statepoint.100.h5'
+sp = openmc.StatePoint(statepoint)
+geometry = sp.summary.geometry
+
+# Close statepoint and summary files to be able to write over them
+sp.summary._f.close()
+sp._f.close()
```

Comment:
Attributes that begin with an underscore are meant to be for internal use only (i.e. users should never need to touch them).  To fix this, make a `close()` method for the `Summary` and `Statepoint` objects that calls the `_f.close()`

Reply:
Ok. I found a __exit__ method for statepoints but I imagine that's still for internal use (called when program exits right ?). Anyway I added a method that calls __exit__, called close()